R/preparedata.R

```diff
@@ -505,6 +505,8 @@ PrepareData <- function(chart.type,
     ###########################################################################
 
 
+    if (!inherits(data, "QTable") && !is.null(attr(data, "span")))
+        attr(data, "span") <- NULL
```

Comment:
Add additional check here because it is fairly easy to generate a hardship error with a span attribute if it is not subscripted properly (which usually does occur because NETs are removed by default)